daemon/info_unix.go

```diff
@@ -217,6 +218,62 @@ func (daemon *Daemon) fillPlatformVersion(v *types.Version) {
 	} else {
 		logrus.Warnf("failed to retrieve %s version: %s", defaultInitBinary, err)
 	}
+
+	daemon.fillRootlessVersion(v)
+}
+
+func (daemon *Daemon) fillRootlessVersion(v *types.Version) {
+	if !rootless.RunningWithRootlessKit() {
```

Comment:
Do you think we should use a `sync.Once` for these? (probably ok for a follow-up)

Reply:
RunningWithRootlessKit already has sync.Once: https://github.com/moby/moby/blob/33c332ad1982019cb9250ed0163ee2ed4955f596/rootless/rootless.go#L20

Calling the RootlessKit API should not be `Once`.